IdentityServer/v6/docs/content/quickstarts/2_interactive.md

```diff
@@ -243,30 +166,185 @@ public static IEnumerable<Client> Clients =>
+client, the client creates its own local cookie. Subsequent requests to the
+client will include this cookie and be authenticated with the default Cookie
+scheme.
+
+After the call to *AddAuthentication*, *AddCookie* adds the handler that can
+process the local cookie.
+
+Finally, *AddOpenIdConnect* is used to configure the handler that performs the
+OpenID Connect protocol. The *Authority* indicates where the trusted token
+service is located. The *ClientId* and the *ClientSecret* identify this client.
+*SaveTokens* is used to persist the tokens in the cookie (as they will be needed
+later).
+
+{{% notice note %}}
+
+This uses the *authorization code* flow with PKCE to connect to the OpenID
+Connect provider. See [here]({{< ref "/fundamentals/clients" >}}) for more
+information on protocol flows.
+
+{{% /notice %}}
+
+### Configure the Pipeline
+Now add *UseAuthentication* to the ASP.NET pipeline in *MvcClient/Program.cs*.
+Also add *RequireAuthorization* to the controller routing to disable anonymous
+access for the entire application. 
+
+```cs
+app.UseRouting();
+app.UseAuthentication();
+app.UseAuthorization();
+
+app.MapControllerRoute(
+    name: "default",
+    pattern: "{controller=Home}/{action=Index}/{id?}")
+    .RequireAuthorization();
+
+```
+
+{{% notice note %}}
+
+You could use the *[Authorize]* attribute instead of this *RequireAuthorization*
+call if you want to specify authorization on a per controller or action basis.
+
+{{% /notice %}}
+
+### Display the Auth Cookie
+
+Modify *Views/Home/Index.cshtml* to display the claims of the user and the
+cookie properties:
+
+```cs
+@using Microsoft.AspNetCore.Authentication
+
+<h2>Claims</h2>
+
+<dl>
+    @foreach (var claim in User.Claims)
+    {
+        <dt>@claim.Type</dt>
+        <dd>@claim.Value</dd>
+    }
+</dl>
+
+<h2>Properties</h2>
+
+<dl>
+    @foreach (var prop in (await Context.AuthenticateAsync()).Properties.Items)
+    {
+        <dt>@prop.Key</dt>
+        <dd>@prop.Value</dd>
+    }
+</dl>
+```
+
+### Configure MvcClient's Port
+Update the client's applicationUrl in
+*MvcClient/Properties/launchSettings.json* to use port 5002.
+
+```json
+{
+  "profiles": {
+    "MvcClient": {
+      "commandName": "Project",
+      "dotnetRunMessages": true,
+      "launchBrowser": true,
+      "applicationUrl": "https://localhost:5002",
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      }
+    }
+  }
+}
+```
 
-Trigger the authentication handshake by navigating to the protected controller action.
-You should see a redirect to the login page of the IdentityServer.
+## Test the client
+Now everything should be in place to log in to *MvcClient* using OIDC. Run
+*IdentityServer* and *MvcClient* and then trigger the authentication handshake by
+navigating to the protected controller action. You should see a redirect to the
+login page in *IdentityServer*.
 
-![](../images/3_login.png)
```

Comment:
I think we should update the MVC client screenshots. Some claims are gone, some have been added (it also says "copyright 2016" ;))